Add origami-version GitHub Action

JakeChampion opened the pull request.

origamiserviceuser commented:
o-utils bundle size difference from 1.1.1 to 10aee315aa4f82576903b3d3bf98476b426edf51
No significant bundle size differences found.

JakeChampion committed:
Add origami-version GitHub Action

When this is merged, we can have automated releases by using the release-type labels "major"/"minor"/"patch" on a pull-request before merging the pull-request.

origamiserviceuser commented:
o-utils bundle size difference from 1.1.1 to c5f54c80bd74cebedc249860f6888740fda49623
No significant bundle size differences found.

JakeChampion commented:
This helps achieve the OKR outlined here https://github.com/Financial-Times/origami/issues/39
